src/server/report/app.js

```diff
@@ -1,35 +1,88 @@
 import './test-result.js';
-import { css, html, LitElement } from 'lit';
+import { css, html, LitElement, nothing } from 'lit';
 import data from './data.js';
 import page from 'page';
 
+const FILTER_STATUS = {
+	ALL: 'All',
+	PASSED: 'Passed',
+	FAILED: 'Failed'
+};
+
 class App extends LitElement {
 	static properties = {
+		_files: { state: true },
 		_filterFile: { state: true },
 		_filterTest: { state: true },
 		_mode: { state: true },
 		_overlay: { state: true }
 	};
 	static styles = [css`
+		.container {
+			display: grid;
+			grid-auto-flow: row;
+			grid-template-areas: "sidebar content";
+			grid-template-columns: 300px auto;
+		}
+		aside {
+			background-color: #fff;
+			border: 1px solid #e6e6e6;
+			box-shadow: 0 0 6px rgba(0,0,0,.07);
+			box-sizing: border-box;
+			grid-area: sidebar;
+			height: 100vh;
+			position: sticky;
+			top: 0;
+			z-index: 10;
+		}
```

Comment:
I stole most of these styles from the Daylight Site. They work for now and we can tweak them later!